index.html

```diff
@@ -4,16 +4,91 @@
 <head>
   <meta charset="utf-8">
 
-  <title>My Resume</title>
+  <title>LV Wood Resume</title>
 
   <link rel="stylesheet" href="css/index.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat|Playfair+Display" rel="stylesheet">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
 
 <body>
-  <h1>My Resume</h1>
+<div class="container">
+    
+<div class="header-nav">
+    <header>
+        <h1>Leslee V. Wood</h1>
+        <nav>
+          <a href="">Experience</a>
+          <a href="">Portfolio</a>
+          <a href="">Education</a>
+          <a href="">Contact</a>
+        </nav>
+    </header>
+</div>
+
+<div>
+  <div class="main-content">
+    <div class="content-section">
+    <section>
+      <h2>Introduction</h2>
+      <p>paragraph bio</p>
+      <h3>Skills</h3>
+      <li>
+        <ul>HTML and CSS</ul>
+        <ul>Adobe Creative Cloud: Illustrator, InDesign, Photoshop, Premiere, Audition</ul>
+        <ul>Site-building tools: Wix, Weebly, Wordpress</ul>
```

Comment:
It looks like you interchanged the `<li>` and `<ul>` tags here. 

`<ul>` is the tag for an unordered list, and `<li>` is for the individual list items (they should be placed inside the `<ul>` tag).